AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Sync/MutationSync/AWSMutationDatabaseAdapter/AWSMutationDatabaseAdapter+MutationEventIngester.swift

```diff
@@ -49,7 +49,7 @@ extension AWSMutationDatabaseAdapter: MutationEventIngester {
         }
 
         MutationEvent.pendingMutationEvents(
-            for: mutationEvent.modelId,
+            forMutationEvent: mutationEvent,
```

Comment:
```suggestion
            forEvent: mutationEvent,
```
nit, or can we just keep it as `for` ?

Reply:
I was initially using `for` for all the methods. 
But considering methods signature:
- `pendingMutationEvents(for: Model...`
- `pendingMutationEvents(for: MutationEvent...`

As `MutationEvent` also confirms to protocol `Model`, if invoke the method with some `MutationEvent`, it will depend on the swift compiler behavior.